### DISCUSSION TITLE
cant find .mo file using OMPython

### DISCUSSION THREAD

**Discussion by ap766** [OpenModelica/OpenModelica#12459] [Q&A]
![image](https://github.com/OpenModelica/OpenModelica/assets/79255079/fa620d99-3235-4ca4-b274-8d15b82c6370)

so I am using OMPython to simulate my model and create the .mat file, but I am getting an Error.
I have used the correct path

**Comment by ap766** [comment]
![image](https://github.com/OpenModelica/OpenModelica/assets/79255079/c0d95177-d331-43a0-a902-dd0ffc434f95)
Here the Batch_Rectifer exports a parameter from Compounds.mo .And Compounds.mo takes it from ChemsepDatabase. Is it perhaps because of that?

**Comment by arun3688** [comment]
@ap766 I guess your model is within the package structure, so your model path should be `/home/XXXXX/package.mo`

**Reply by ap766** [reply]
![image](https://github.com/OpenModelica/OpenModelica/assets/79255079/8f4a6b3a-1f44-4c32-b314-d6828b54d1fd)
My package.mo specifies modules as the package

**Reply by arun3688** [reply]
@how do you load your model in `OMEdit`

**Reply by ap766** [reply]
![image](https://github.com/OpenModelica/OpenModelica/assets/79255079/74182938-8def-4ffd-add7-1a58a23638f6)
I go to Load Library from FIle and select Modules from my FIle Explorer.

**Reply by ap766** [reply]
So as per the docs, I have included the package name in the second arguement
![image](https://github.com/OpenModelica/OpenModelica/assets/79255079/95c7a1ff-6c06-48e8-866a-71e8561b9e14)

However still getting an error
![image](https://github.com/OpenModelica/OpenModelica/assets/79255079/44191229-2854-4826-abe7-f9e7704a34c8)

**Reply by arun3688** [reply]
@ap766 Please adde package.mo to your path `/home/anisha7/Desktop/OM/Modules/package.mo` and it should work

**Reply by ap766** [reply]
![image](https://github.com/OpenModelica/OpenModelica/assets/79255079/c4f98237-e761-4860-9c46-97e8bfc36c32)
Its giving the error that it doesn't exist

**Reply by ap766** [reply]
![image](https://github.com/OpenModelica/OpenModelica/assets/79255079/1e091583-4a81-492c-9618-2517d4125510)

I removed the Modules and tried as well 
Basically like this

> 
> from OMPython import ModelicaSystem
> 
> model_path2 = "/home/anisha7/Desktop/OM/Modules/package.mo/"
> mod = ModelicaSystem(model_path2 + "Batch_Rectifier.mo", "Batch_Rectifier")
> mod.simulate(resultfile="Batch_Rectifier.mat")
>

**Reply by arun3688** [reply]
can you please send your whole modules as a zip file

**Reply by ap766** [reply]
[Modules.zip](https://github.com/OpenModelica/OpenModelica/files/15415916/Modules.zip)

**Comment by arun3688** [comment]
@ap766 The following code works

```
>>> from OMPython import ModelicaSystem
>>> mod = ModelicaSystem("C:/OPENMODELICAGIT/matlab/Modules/package.mo", "Modules.Batch_Rectifier")
```
